sources/web/datalab/polymer/components/resizable-divider/resizable-divider.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Resizable divider element for Datalab.
+ * This element is a container for two elements with a vertical resize divider.
+ */
+class ResizableDividerElement extends Polymer.Element {
+
+  /**
+   * Minimum pane width in pixels. Defaults to 50(px);
+   */
+  public minimumWidthPx: number;
+
+  /**
+   * Initial position of the divider in percentage. Defaults to 50(%);
+   */
+  public initDividerPosition: number;
+
+  private _boundMouseDownHandler: EventListenerOrEventListenerObject;
+  private _boundMouseupHandler: EventListenerOrEventListenerObject;
+  private _boundMouseMoveHandler: EventListenerOrEventListenerObject;
+  static get is() { return 'resizable-divider'; }
+
+  static get properties() {
+    return {
+      initDividerPosition: {
```

Comment:
If this parameter only accepts a percent, it should probably be called initialDividerPercent to make that clear. Or maybe it should have "Left" in the name somewhere to indicate that this is the size of the left panel (otherwise, we are assuming that everyone will think left-to-right percent). In line with the comment a few lines down, maybe it could be "initialLeftPaneWidth" and accept either "70%" or "300px" as a value. Or allow "initialRightPaneWidth" instead. I'll let you decide how much of that is worth doing right now, but if you decide to minimize the work, I'll stick to my suggestion to at least change the name from "Position" to "Percent".